docs/configuration.md

```diff
@@ -0,0 +1,127 @@
+# Configuration
+
+## Configuration methods
```

Comment:
@MrAlias Should we add content here akin the [.NET docs](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/config.md#configuration-methods)? If so, can you provide rough content for me to add, or point to relevant resources?

Reply:
The environmental variables listed below are used to configure the auto-instrumentation binary: https://github.com/open-telemetry/opentelemetry-go-instrumentation/blob/main/cli/main.go.

For the binary one can also set the logging level via `-logLevel` and enable recording telemetry from the OpenTelemetry default global implementation the global via `global-impl` CLI flags.

Users can also chose build their own Go automatic instrumentation binaries using https://pkg.go.dev/go.opentelemetry.io/auto and add support for more/other configurations.

Reply:
Should `global-impl` have a dash at the beginning (`-global-impl`)?

Reply:
Yes, it should.